src/controllers/unreal.rs

```diff
@@ -111,6 +115,17 @@ enum ControlEvent {
     TranslateEye(Vec2),
 }
 
+fn on_controller_enabled_changed(
```

Comment:
I didn't like duplicating this system on each controller. However, as long as Bevy doesn't support dynamic (component types in) queries, I think there is no other way? Or I missed something else that could have been done better!

Reply:
I think you could use a generic system for this. Replace `UnrealCameraController` with a generic type parameter.